morango/migrations/0010_auto_20170807_0123.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2017-08-07 01:23
+from __future__ import unicode_literals
+
+import datetime
+from django.db import migrations, models
+import django.utils.timezone
+from django.utils.timezone import utc
+import morango.utils.uuids
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('morango', '0009_auto_20170731_2142'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Nonce',
+            fields=[
+                ('id', morango.utils.uuids.UUIDField(editable=False, primary_key=True, serialize=False)),
+                ('timestamp', models.DateTimeField(default=django.utils.timezone.now)),
+                ('ip', models.CharField(blank=True, max_length=100)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.RemoveField(
+            model_name='syncsession',
+            name='connection_params',
+        ),
+        migrations.RemoveField(
+            model_name='transfersession',
+            name='chunksize',
+        ),
+        migrations.RemoveField(
+            model_name='transfersession',
+            name='records_remaining',
+        ),
+        migrations.AddField(
+            model_name='buffer',
+            name='source_id',
+            field=models.CharField(default=datetime.datetime(2017, 8, 7, 1, 22, 33, 701906, tzinfo=utc), max_length=96),
```

Comment:
MAGIC!

I think it required a default value, and it asked me, and it told me you can use `datetime.datetime()`, so I... did? :D

My plan is to delete all these new migrations and create a fresh slate before we integrate into Kolibri.